provider/pkg/clients/unstructured.go

```diff
@@ -65,22 +65,35 @@ func ToUnstructured(object metav1.Object) (*unstructured.Unstructured, error) {
 // This process normalizes semantically-equivalent resources into an identical output, which is important for diffing.
 // If the scheme is not defined, then return the original resource.
 func Normalize(uns *unstructured.Unstructured) (*unstructured.Unstructured, error) {
+	var result *unstructured.Unstructured
+
 	if IsCRD(uns) {
```

Comment:
nit: preference for switch case here, but will follow up in a later PR/discussion to get this fix out ASAP.